apps/codesandbox-react-next-template/tsconfig.json

```diff
@@ -11,10 +11,13 @@
     "forceConsistentCasingInFileNames": true,
     "moduleResolution": "node",
     "preserveConstEnums": true,
-    "strict": true,
+    "strictNullChecks": true,
```

Comment:
I'm curious, what issues were you hitting with `strict`?

Reply:
https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=112673&view=logs&j=258ec178-2d8b-5611-7b9b-60c5c95dae55&t=e99f753c-3f64-5289-3575-c1a650af1617

Reply:
Ah okay, I forgot that `strict` implies `strictFunctionTypes`

Reply:
The underlying issue is that this tsconfig's strictness should be in line with the underlying projects it's building, rather than stricter. Otherwise, the source project builds fine, and you get these spurious errors downstream.